Protacon.RxMq.AzureServiceBusLegacy/Topic/AzureBusTopicSubscriber.cs

```diff
@@ -81,30 +83,42 @@ internal Binding(MessagingFactory messagingFactory, NamespaceManager namespaceMa
                 }, _options);
             }
 
-            private static void MakeSureSubscriptionExists(NamespaceManager namespaceManager, AzureTopicMqSettings settings,
-                string topicPath, string subscriptionName)
+            private void MakeSureSubscriptionExists(NamespaceManager namespaceManager, AzureTopicMqSettings settings,
+                string topicPath, string subscriptionName, bool removePrevious = false)
             {
+                var subscriptionDescription = new SubscriptionDescription(topicPath, subscriptionName);
                 if (!namespaceManager.SubscriptionExists(topicPath, subscriptionName))
                 {
-                    var subscriptionDescription = new SubscriptionDescription(topicPath, subscriptionName);
                     namespaceManager.CreateSubscription(settings.SubscriptionBuilderConfig(subscriptionDescription, typeof(T)));
+                    _logMessage($"MakeSureSubscriptionExists: Created subscription: {subscriptionName}");
```

Comment:
classname/-prefix

Reply:
kaikkiin

Reply:
I will follow general way we log from RxMq. Will not add classname prefix.